src/framework/theme/components/layout/layout.component.spec.ts

```diff
@@ -66,7 +66,7 @@ describe('NbLayoutComponent', () => {
     // }));
     //
     // it('should add left padding to layout container in RTL mode when blocking scroll', fakeAsync(() => {
-    //   const layoutDirectionService: NbLayoutDirectionService = TestBed.get(NbLayoutDirectionService);
```

Comment:
Would you mind removing entire comment?) We have a version control system, we don't need to comment unused code.

Reply:
https://github.com/angular/components/pull/15440